lib/generamba/cli/gen_command.rb

```diff
@@ -58,5 +61,24 @@ def gen(module_name, template_name)
       generator.generate_module(module_name, code_module, template)
     end
 
+    no_commands {
+      def print_parameters_in_table(parameters, module_name)
+        params = parameters.clone
+
+        templates = []
+
+        params["templates"].each do |param|
```

Comment:
We may occasionally add another array attributes in the future (e.g. currently it's possible to specify custom catalogs in this way). What do you think, maybe we should move the logic of converting all arrays into string down to the PrintTable.print method itself? You can traverse through all of the keys and convert all discovered arrays.